packages/core/src/internal/decorators/i18n.spec.ts

```diff
@@ -24,6 +30,18 @@ class TestI18nElement extends LitElement {
   }
 }
 
+/** @element test-alert-18n-element */
+@customElement('test-alert-18n-element')
+class TestAlertI18nElement extends LitElement {
```

Comment:
I recreated the situation of @coryrylan 's stackblitz of the i18n not properly merging overrides with the i18n keys.